apps/passport-server/.env.local.example

```diff
@@ -73,5 +73,15 @@ SERVER_EDDSA_PRIVATE_KEY=
 TELEGRAM_BOT_TOKEN=
 TELEGRAM_BOT_START_DELAY_MS=0
 
-# Disable Pretix sync for local development
-PRETIX_SYNC_DISABLED=TRUE
+# Disable Pretix sync for local development. 
+# Now that we use Pretix sync to load the dev event + tickets, we want to keep Pretix sync enabled.
+PRETIX_SYNC_DISABLED=
+
+# Must match format of `https://pretix.edu/api/v1/organizers/<org_id>
```

Comment:
Technically, it doesn't need to match the exact format, because it could be a self-hosted Pretix instance (e.g. the EF will have this). For example in staging, we have https://ticketh.xyz/api/v1/organizers/0xparc

```suggestion
# Must be a valid URL prefix, e.g. `https://ticketh.xyz/api/v1/organizers/<org-id>`
```